docs/release.md

```diff
@@ -0,0 +1,21 @@
+# Appium Java Client Release Procedure
+
+This document describes the process of releasing this client to the Maven repository.
+Its target auditory is project maintainers.
+
+## Release Steps
+
+1. Update the [Changelog](../CHANGELOG.md) for the given version based on previous commits.
```

Comment:
We also need to bump the version here https://github.com/appium/java-client/blob/master/gradle.properties

Reply:
added